pyproject.toml

```
@@ -28,7 +28,8 @@ classifiers = [
 python = "^3.9"
 fastapi = "^0.68.0"
 aiohttp = "^3"
-python-jose = {extras = ["cryptography"], version = "^3.3.0"}
+cryptography = "^35.0.0"
+python-jose = "^3.3.0"
```

Perfect, cryptography should be like that.Please change the python-jose back to specifying extras.
```suggestion
python-jose = {extras = ["cryptography"], version = "^3.3.0"}
```
And then please do a `poetry update` for the lockfile to update. :blush: